addons/packages/fluxcd-kustomize-controller/0.24.4/package.yaml

```diff
@@ -11,7 +11,7 @@ spec:
     spec:
       fetch:
         - imgpkgBundle:
-            image: projects.registry.vmware.com/tce/fluxcd-kustomize-controller-bundle@sha256:424e08bc92768bbb179a8ba09c0062619dabcba1614736bbd69c025422bbd2f4
+            image: projects.registry.vmware.com/tce/fluxcd-kustomize-controller-bundle:0.24.4-update.1
```

Comment:
@chaudharyalka You don't want to use a tag here. On an OCI Registry, the tag can potentially be changed to another image/bundle, so there is no guarantee of what you're going to actually be referring to. By using the sha256, you're guaranteed to get the correct image.

Can you switch this back to using the sha?

Reply:
@seemiller Done.